edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Edgelet.Docker/CombinedEdgeletConfigProvider.cs

```diff
@@ -62,6 +67,31 @@ static string BindPath(Uri uri)
                 : uri.AbsolutePath;
         }
 
+        static void InjectEdgeAgentLabels(IModule module, CreateContainerParameters createOptions)
```

Comment:
do you know what is the difference between CreateContainerParameters.Env (IList<string>) and IModule.Env (IDictionary<string, EnvVal>)?

Reply:
Potentially the same information, just different structures for different parts of the pipeline. `CreateContainerParameters` is Docker-specific, while `IModule` is module type-agnostic.